common.php

```diff
@@ -48,10 +48,33 @@ function crash($errorCode, $data = null) {
 	define("SUMMER", 1);
 	define("FALL", 2);
 
+	$season_name = [
```

Comment:
I would make this a constant since it should never change:
`const SEASON_NAME = ["Spring", "Summer", "Fall"];`